interface.go

```diff
@@ -175,6 +175,16 @@ type JobRetryInfo struct {
 	// will be allowed to run 3 times at most. If unset, the default maximum
 	// attempts is 10.
 	MaxAttempts int `bson:"max_attempts,omitempty" json:"max_attempts,omitempty" yaml:"max_attempts,omitempty"`
+	// DispatchBy reflects the amount of time (relative to when the job is
+	// retried) that the retried job has to dispatch for execution. If this
+	// deadline elapses, the job will not run. This is analogous to
+	// (JobTimeInfo).DispatchBy.
+	DispatchBy time.Duration `bson:"dispatch_by,omitempty" json:"dispatch_by,omitempty" yaml:"dispatch_by,omitempty"`
```

Comment:
So the retry job will not inherit the `DispatchBy` and `WaitUntil` settings of the original job?

Reply:
It inherits the original value from the job if it's not specified here.